core/lib/spree/event.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require_relative 'event/adapters/active_support_notifications'
+require_relative 'event/configuration'
+
+module Spree
+  module Event
+    POSTFIX = '.spree'
```

Comment:
I always see `SUFFIX` for this kind of things. Anyway, what about making this configurable?

Reply:
👍 I think making this configurable is a nice addition

Regarding the naming, I think [both are fine](https://english.stackexchange.com/questions/81263/postfix-or-suffix) but yes, probably `suffix` is more common